cmd/kubectl.lima

```diff
@@ -33,5 +33,5 @@ if [ -n "$KUBECTL" ]; then
   exec "$KUBECTL" "$@"
 else
   export LIMA_INSTANCE
-  exec lima sudo kubectl "$@"
+  exec lima kubectl "$@"
```

Comment:
Is there a reason to remove the `sudo`? This will make existing instances created by previous Lima releases incompatible with the new script.

Since the kubeconfig still exists in both places, the existing code would continue to work, no?

Reply:
It should work now, similar to using `sudo docker` even if you are already part of the root group

Reply:
Yes, that's why you should leave the `sudo` in the script it, to avoid breaking it for older instances:

* User has Lima 0.19 and creates a `k8s` instance, which works fine with `kubectl.lima`.
* User upgrades to Lima 0.20 but keeps existing `k8s` instance.
* `kubectl.lima` no longer works because `${LIMA_CIDATA_HOME}/.kube/config` does not exist

The provisioning script is guarded with

```shell
test -e /etc/kubernetes/admin.conf && exit 0
```

So it won't be executed when you restart the existing instance with 0.20 and the user-mode kubeadmin will never get created.

Alll this can be avoided by leaving the `sudo` in `kubectl.lima`. Or if there is some advantage to removing it, then the provisioning script needs to be changed so that `${LIMA_CIDATA_HOME}/.kube/config` is created on each boot to migrate existing instances over.

Reply:
> Or if there is some advantage to removing it, then the provisioning script needs to be changed so that ${LIMA_CIDATA_HOME}/.kube/config is created on each boot to migrate existing instances over.

I'm sorry; that was a thinko on my part. The old instance will never see the new provisioning scripts; so `kubectl.lima` needs to continue to work with the old one if we don't want to break backwards compatibility (and I think we shouldn't).

So I think leaving the `sudo` in the `kubectl.lima` script is the right way to deal with this.

Sorry for the confusion!